pom.xml

```diff
@@ -927,15 +873,4 @@ flexible messaging model and an intuitive client API.</description>
       <url>http://yahoo.bintray.com/maven</url>
     </repository>
   </repositories>
-
-  <distributionManagement>
-    <snapshotRepository>
```

Comment:
should we add `Apache Nexus repo url` in `distributionManagement`.? How would travis know the repo url? does `Papache-release` indicates repo-url?